src/Contracts/Google2FA.php

```diff
@@ -82,11 +82,24 @@ public function removeInvalidChars($string);
 	/**
 	 * Creates a Google QR code url.
 	 *
-	 * @param $company
-	 * @param $holder
-	 * @param $secret
+	 * @param string $company
+	 * @param string $holder
+	 * @param string $secret
+	 * @param integer $size
 	 * @return string
 	 */
-	public function getQRCodeGoogleUrl($company, $holder, $secret);
+	public function getQRCodeGoogleUrl($company, $holder, $secret, $size = 200);
```

Comment:
Kind of feel like the Contract is a bit heavy on methods anyways, don't you think? Seem a bit implementation specific, instead of useful for end-users.